lib/linguist/heuristics.rb

```diff
@@ -473,5 +473,14 @@ def call(data)
         Language["XML"]
       end
     end
+
+    disambiguate ".w" do |data|
+      if (data.include?("&ANALYZE-SUSPEND _UIB-CODE-BLOCK _CUSTOM _DEFINITIONS C-Win"))
+        Language["OpenEdge ABL"]
+      elseif /^@(<|\w+\.)/.match(data)
```

Comment:
Doh!! 🤦‍♂️  I completely missed this should be `elsif`... not `elseif` (teaches me for working in multiple languages in one day)